docs/_source/_common/snippets/training/token-classification/transformers.md

```diff
@@ -0,0 +1,19 @@
+---
+title: This is how to train your Token Classifier dataset using spaCy
```

Comment:
Hi! I know this is still WIP, just a minor nit here :hugs:

```suggestion
title: This is how to train your Token Classifier dataset using Transformers
```

Reply:
Hi @alvarobartt, that was correct. I believe I already covered this in my previous commits. Thank you for the help though:)